docs/rules/key-spacing.md

```diff
@@ -162,6 +162,35 @@ var obj = {
 };
 ```
 
+#### Fine-grained control
+
+You can specify these options separately for single-line and multi-line configurations by organizing the options this way:
+
+```js
+"key-spacing": [2, {
+        "singleLine": {
```

Comment:
So if I have configuration like this: `key-spacing: [2, { beforeColon: true, simpleLine: { beforeColon: false } } ]` based on the code it will still enforce spacing before colon for single line, right?